lib/php/rendition-factory/src/Transformer/Document/PdfToImageTransformerModule.php

```diff
@@ -39,7 +39,32 @@ public function buildConfiguration(NodeBuilder $builder): void
         // @formatter:off
         $builder
             ->arrayNode('options')
-            ->ignoreExtraKeys(false)
+                ->children()
+                    ->scalarNode('extension')
+                        ->info('Output image extension: jpg, jpeg, png, or webp')
+                        ->defaultValue('jpeg')
+                    ->end()
+                    ->integerNode('resolution')
+                        ->info('Resolution of the output image in dpi')
+                        ->defaultValue(300)
+                    ->end()
+                    ->integerNode('quality')
+                        ->info('Quality of the output image, from 0 to 100')
+                        ->defaultValue(100)
+                    ->end()
+                    ->arrayNode('size')
+                        ->info('Size of the output image, [width, height] in pixels')
+                        ->children()
+                            ->integerNode(0)
+                                ->info('Width of the output image in pixels')
+                                ->example(150)
+                            ->end()
+                            ->integerNode(1)
+                                ->info('Height of the output image in pixels')
+                                ->example(100)
+                            ->end()
+                    ->end()
```

Comment:
Why not just using `integerNode('width')` and same for height?
This would simplify the documentation. Also if values are required you need to add `isRequired()`  chained call.

Reply:
if we use `integerNode('width')` we cannot  define config like size: [300,300]
we must do `size: width: 300`

and all size in other filter like module imagine is like that https://symfony.com/bundles/LiipImagineBundle/current/filters/sizing.html#thumbnails